### ISSUE TITLE
Reproducing Results for Finetuning ImageMol on GPCR assays and Sars-CoV2 assays

### ISSUE THREAD

**Issue by DhanshreeA** [HongxinXiang/ImageMol#6]
Hi! 
I have been trying to fine tune ImageMol on SARS-CoV-2 and GPCR assays but I have not been able to get the same rocauc score as mentioned in the paper's results. Here are some of the hyperparam values that I have tried. along with their results: 

**3CL enzymatic activity**

- lr: 0.5, batch 128, epochs 20 with `final results: highest_valid: 0.685, final_train: 0.532, final_test: 0.542`
- lr: 0.1, batch 32, epochs 20 with `final results: highest_valid: 0.586, final_train: 0.489, final_test: 0.240` 
- lr: 0.01, batch 32, epochs 30 with `final results: highest_valid: 0.551, final_train: 0.956, final_test: 0.677`

It's a bit odd  that there's a downward trend within the validation score with decreasing the learning rate. Could you please provide me with the exact choice of hyperparameter values for reproducing the paper's results across all 13 SARS-CoV-2 and the 10 GPCR assays? It's a tall ask but if you could share the fine tuned models, that would be fantastic!

I am a volunteer at https://github.com/ersilia-os/ , a non profit,  where we are building data science tools to aid drug discovery in low resourced countries. We are very interested in incorporating ImageMol fine tuned on these assays in our model hub because this will be immensely valuable for the community, and we value any help you could offer on this front.

**Comment by HongxinXiang** [comment]
Hi Dhanshree! 
Thank you for your interest in our work. I'm dealing with a deadline currently. Give me some time and I'll share our fine-tuned models.

**Comment by DhanshreeA** [comment]
Hi @HongxinXiang thank you for your time and support. I am following up regarding the fine tuned models. Please let me know if I can help in anyway. Thank you!

**Comment by HongxinXiang** [comment]
Hi @DhanshreeA Sorry for the wait. You can access the fine-tuned models via the link below:

13 SARS-CoV-2: https://drive.google.com/file/d/15h6dZDW4ntdVvmQGOh0RTE4-uq7wZTFk/view?usp=sharing

10 GPCR: https://drive.google.com/file/d/1FIrc4hNeZi_xuLvMJ-5n6DisfsTfPVAy/view?usp=sharing

**Comment by DhanshreeA** [comment]
Thank you so much @HongxinXiang! This is really helpful. I have downloaded these checkpoints and I am now working on adding these models to our hub. You are very welcome to view our (currently in progress) work here :smile: : https://github.com/ersilia-os/ersilia/issues/572

**Comment by DhanshreeA** [comment]
Thank you so much @HongxinXiang! This is really helpful. I have downloaded these checkpoints and I am now working on adding these models to our hub. You are very welcome to view our (currently in progress) work here :smile: : https://github.com/ersilia-os/ersilia/issues/572
